### PR TITLE
add design and js for switch theme

### PR DESCRIPTION
**Pull request by LrxGaelle**

### CODE REVIEW

File: doc_sphinx/_static/js/main.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const toggleSwitch = document.querySelector('#themeSwitch');
+const html = document.querySelector('html');
```
Comment (benjaminaudet) [inline doc_sphinx/_static/js/main.js]: Prefix your variable with '$'  when its a DOM Element => Habitabilty purposes

File: doc_sphinx/_templates/home.html [hunk 2]
```diff
@@ -11,8 +11,8 @@ <h1 class="text-center">Documentation</h1>
           <div class="row">
             <div class="col-12">
               <div class="flex-container_equal">
-                <div class="card-horizontal text-center">
-                  <div class="card-horizontal_content">
+                <div class="card-vertical text-center">
+                  <div class="card-vertical_content">
```
Comment (benjaminaudet) [inline doc_sphinx/_templates/home.html]: Avoid underscore in HTML class and id => Convention purposes

Comment (LrxGaelle) [reply doc_sphinx/_templates/home.html]: The use of the underscore is to use the convention already implemented on all projects

Comment (benjaminaudet) [reply doc_sphinx/_templates/home.html]: :(

**Pull request closed without merging**